schema.graphql

```diff
@@ -85,3 +85,9 @@ type BatchParameter @entity {
   name: String!
   value: String!
 }
+
+type SetController @entity {
+  id: ID! #blockHeight-eventIdx
+  stash: String!
+  controller: String!
```

Comment:
extrinsic: Extrinsic!

would be nice to have the extrinsic entity querable from that entity

Reply:
I'm not sure what you mean. What's the advantage of that? I'm only interested in the stash-controller fields.